### PR TITLE
mon/OSDMonitor: fix dividing by zero in OSDUtilizationDumper

### PR DESCRIPTION
**Pull request by LiumxNL**
Signed-off-by: Mingxin Liu <mingxin@xsky.com>

### CODE REVIEW

File: src/mon/OSDMonitor.cc [hunk 1]
```diff
@@ -641,7 +641,10 @@ class OSDUtilizationDumper : public CrushTreeDumper::Dumper<F> {
     int64_t kb = 0, kb_used = 0, kb_avail = 0;
     double util = 0;
     if (get_bucket_utilization(qi.id, &kb, &kb_used, &kb_avail))
+    if (kb_used) {
+      assert(kb > 0);
```
Comment (tchaikov) [inline src/mon/OSDMonitor.cc]: this does not fix the divide by zero issue.  if kb is 0, monitor crashes as before.

**Comment by Daveweco** [comment]
LGTM

**Comment by LiumxNL** [comment]
@tchaikov updated

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-02-27_04:13:29-rados-wip-kefu-testing---basic-smithi

**Pull request merged**